README.md

```diff
@@ -189,20 +193,24 @@ and ensure it says "Using **development build**". If it says "Using **Expo Go**"
 
 You can then press `a`, `i`, or `w` to run Android, iOS, or Web. You can run any or all of them at the same time. It will open the emulator or browser for you as needed.
 
-### Debugging with React DevTools
-
-You can run React Devtools with `npm run react-devtools`. This will start the DevTools up which will be ready to open with Metro's in-app dev menu. In running Metro app make sure to hit press 'd' to open the dev menu in React DevTools.
-Alternatively, if you start up the Emulator after React DevTools is running it'll automatically watch the emulator. If for some reason it doesn't watch it, press 'd' on the terminal that opens for the Metro Dev server.
-
-While using the emulator if you wish to use the element inspector, run the `adb shell input keyevent 82` in the terminal.
+Start the API server in a separate terminal with:
 
-### Running the local place-my-order-api server for the android emulator
+```
+npm run api
+```
 
 By default, the Android emulator won't be able to access your localhost server.
```

Comment:
Added the command to run API server above and moved this paragraph to the "Running" section as this seems like a necessary step to do for android emulators.